recipes/novoalign/meta.yaml

```diff
@@ -26,6 +26,8 @@ requirements:
   run:
     # curl is needed to access Novocraft's website during package install -- see post-link.sh
     - curl
+    # Novoalign 4.03.04 pre-compiled binary needs libsmartcols from util-linux.
+    - util-linux
```

Comment:
`strace -f -e openat novoalign --version 2>&1 | grep -F .so`
includes
`[pid 393] openat(AT_FDCWD, "/lib/x86_64-linux-gnu/libsmartcols.so.1", O_RDONLY|O_CLOEXEC) = 3`
which means it needs `libsmartcols` which is part of the `util-linux` package.

Reply:
And if we look at the `strace` lines before the above one, we can see that it comes from `lscpu` (also from `util-linux`):
```sh
# strace -f novoalign --version 2>&1 | grep -FB10 .so | grep -A10 exec
[pid   704] execve("/bin/sh", ["sh", "-c", "lscpu -ap 2>/dev/null"], 0x7fff32e27480 /* 14 vars */ <unfinished ...>
[pid   703] read(3,  <unfinished ...>
[pid   704] <... execve resumed> )      = 0
[pid   704] brk(NULL)                   = 0x55c086eda000
[pid   704] access("/etc/ld.so.preload", R_OK) = -1 ENOENT (No such file or directory)
[pid   704] openat(AT_FDCWD, "/etc/ld.so.cache", O_RDONLY|O_CLOEXEC) = 3
[pid   704] fstat(3, {st_mode=S_IFREG|0644, st_size=16451, ...}) = 0
[pid   704] mmap(NULL, 16451, PROT_READ, MAP_PRIVATE, 3, 0) = 0x7fc13b466000
[pid   704] close(3)                    = 0
[pid   704] openat(AT_FDCWD, "/lib/x86_64-linux-gnu/libc.so.6", O_RDONLY|O_CLOEXEC) = 3
--
[pid   704] stat("/usr/local/bin/lscpu", 0x7ffc18dd71e0) = -1 ENOENT (No such file or directory)
[pid   704] stat("/usr/local/sbin/lscpu", 0x7ffc18dd71e0) = -1 ENOENT (No such file or directory)
--
[pid   705] execve("/usr/bin/lscpu", ["lscpu", "-ap"], 0x55c086d30c18 /* 14 vars */) = 0
[pid   705] brk(NULL)                   = 0x55d55b7e7000
[pid   705] access("/etc/ld.so.preload", R_OK) = -1 ENOENT (No such file or directory)
[pid   705] openat(AT_FDCWD, "/etc/ld.so.cache", O_RDONLY|O_CLOEXEC) = 3
[pid   705] fstat(3, {st_mode=S_IFREG|0644, st_size=16451, ...}) = 0
[pid   705] mmap(NULL, 16451, PROT_READ, MAP_PRIVATE, 3, 0) = 0x7f6fd728d000
[pid   705] close(3)                    = 0
[pid   705] openat(AT_FDCWD, "/lib/x86_64-linux-gnu/libsmartcols.so.1", O_RDONLY|O_CLOEXEC) = 3
[pid   705] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0P\217\0\0\0\0\0\0"..., 832) = 832
[pid   705] fstat(3, {st_mode=S_IFREG|0644, st_size=223360, ...}) = 0
[pid   705] mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f6fd728b000
```